microstream-rest/src/test/groovy/io/micronaut/microstream/rest/Towns.groovy

```diff
@@ -0,0 +1,8 @@
+package io.micronaut.microstream.rest
+
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+class Towns {
+    List<String> towns = []
```

Comment:
use nullability annotations to return type

Reply:
Done, but I don't see the value